gateway-service/src/main/resources/gateway-api-doc.json

```diff
@@ -164,11 +168,38 @@
                     }
                 }
             }
+        },
+        "/version": {
+            "get": {
+                "tags": [
+                    "Diagnostic"
+                ],
+                "summary": "Return version information of API Mediation Layer and Zowe",
+                "description": "Use the `/version` API to get the version information of API Mediation Layer and Zowe.\nThe version information includes version, build number and commit hash.\nIn the response can be only API ML version information or API ML and Zowe version information, this depends on API ML installed as part of Zowe build or as standalone application.",
+                "operationId": "VersionInfoUsingGET",
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/VersionInfo"
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "servers": [
+                {
+                    "url": "/"
```

Comment:
![image](https://user-images.githubusercontent.com/6776217/76196038-ccd4e500-61e9-11ea-8904-b4c4027b61c2.png)
Calling the endpoint for version as documented is timing out and throwing zuul exceptions on gw
implies URL https://localhost:10010/api/v1/gateway/version
Please help me if i'm interpreting the information on the apidoc incorrectly

Reply:
Not actual